.github/workflows/initializr-build.yml

```diff
@@ -72,3 +72,43 @@ jobs:
           imageTag=(`./gradlew casVersion --q`)
           echo "Pushing Docker image with tag $imageTag"
           docker push apereo/cas-initializr:"$imageTag"
+  helm:
+    needs: [build]
+    runs-on: ubuntu-latest
```

Comment:
Could we add support for Windows and MACOS here?

Reply:
Running the helm installation on windows or MacOS would require a linux VM with kubernetes, which I don't imagine is that could be done in cicd. I have run 32-bit linux VMs inside windows VMs before and not sure what is possible these days but at the end of the day it would still be getting installed on linux (possibly using a windows version of helm but I would expect that to behave the same as the linux version). To test this out locally I used k3s on a linux VM (windows hyper-v/ubuntu) but I also have Docker Desktop and it should work on that if kubernetes is enabled (haven't tried yet but I have previous installed the same ingress controller and another cas helm chart before when I was running load tests against a two server cas cluster).

Reply:
Makes sense, thanks. On MacOS, I typically use minikube and VirtualBox; I imagine it'd be rather complicated to set all that for CI/CD and probably not worth it at this point. We can worry about it later.